src/main/scala/is/hail/annotations/UnsafeRow.scala

```diff
@@ -301,3 +301,14 @@ class UnsafeRow(var t: TBaseStruct,
     }
   }
 }
+
+class KeyedRow(var row: Row, keyFields: Array[Int]) extends Row {
+  def this(row: Row) = this(row, Array.range(0, row.size))
```

Comment:
this constructor doesn't seem used, can we delete it?

Reply:
Oh, yes. That was from when I didn't have `KeyedRow` extending `Row`.